### PR TITLE
chore:Replaced the community actions with the scripts

### PR DESCRIPTION
**Pull request by ravali-rimmalapudi**
# Fixes #

A short description of what this PR does.

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-cli/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please file a [support ticket](https://twilio.com/help/contact), or create a GitHub Issue in this repository.

### CODE REVIEW

**Comment by shrutiburman** [comment]
Related PR: https://github.com/twilio/twilio-cli-core/pull/155

**Review by shamantraghav — APPROVED** [review]
LGTM!

**Review by shrutiburman — APPROVED** [review]
Good job!

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -63,52 +63,62 @@ jobs:
...
-          GITHUB_TOKEN: ${{ github.token }}
-          TAG_NAME: ${{needs.release.outputs.tag-name}}
-        with:
-          is_append_body: true
-          body: ${{ github.event.inputs.change-log }}
+          if: ${{steps.semantic-release.outputs.tag-name != ''}}
+          run: node .github/scripts/update-release.js
+          env:
+            GITHUB_TOKEN: ${{ github.token }}
+            TAG_NAME: ${{steps.semantic-release.outputs.tag-name}}
```
Comment (shrutiburman) [inline .github/workflows/release.yml:71]: Is the space after `${{ `  optional?

**Pull request merged**